### ISSUE TITLE
"Could not authenticate" message , but registry returning 200 HTTP Status Code

### ISSUE THREAD

**Issue by kbraak** [gbif/ipt#316]
```
What steps will reproduce the problem?
1. Manage Resources
2. New Metadata Resource
3. In Basic Metadata> I fill (required fields) Title, Res Language, Type,
Contact Name, Contact Email and Description
4. In Resource Creator> I fill (required fields) FirstName, LastName,
Email, Country
5. In Metadata Provider> I fill (required fields) FirstName, LastName,
Email, Country
6. Hit Save
7. Get a "Could not authenticate to GBIF registry. Please verify your
organisation key and password."

  - On the registry logs I get:

[gbif-gbrds] INFO [TP-Processor6] OrganisationController.show(438) | show()
- GET organisation:
http://gbrdsdev.gbif.org/registry/organisation/3780d048-8e18-4c0c-afcd-cb6389df56de
[gbif-gbrds] DEBUG [TP-Processor6] OrganisationController.show(443) |
Param: op: login
[gbif-gbrds] DEBUG [TP-Processor6] OrganisationController.show(452) |
show() - OP code
[gbif-gbrds] DEBUG [TP-Processor6] OrganisationController.show(468) |
show() - OP code=login with auth header=Basic
Mzc4MGQwNDgtOGUxOC00YzBjLWFmY2QtY2I2Mzg5ZGY1NmRlOnBhc3N3b3Jk
[gbif-gbrds] DEBUG [TP-Processor6] OrganisationController.show(496) |
op=login successful: Returning HTTP 200 Status Code

So basically, the Registry returns a HTTP 200 Status Code indicating a
success. Are you expecting any other Status Code or any value being
returned, that I perhaps might have forgotten?


What version of the provider software are you using? The version should be
displayed in the footer of any page.

ipt.gbif.org / 1.0RC4-PREVIEW



```

Original issue reported on code.google.com by `josecuadra` on 2010-05-12 11:52:46
